src/main/bluesky/parser.py

```diff
@@ -37,53 +37,60 @@ def calculate_bearing(lat1, lon1, lat2, lon2):
     return round(0 if np.isnan(bearing) else bearing, 2)
 
 
-def write_scene_file(filename, combined_df):
+def write_scene_file(filename, combined_df, total_time):
     with open(filename, 'w') as file:
+        current_time = 0
+        time_step = int(total_time) / len(combined_df)
+
         for index, row in combined_df.iterrows():
             scenetext = ""
             lat, lon = get_airport_info(row["DEST"], "lat"), get_airport_info(row["DEST"], "lon")
             ehamLat, ehamLon = 52.309, 4.764
             stack = row.get("STACK", None)
-            # check if stack is not empty and set stack and brng
+
+            # Calculate time stamp
+            current_time += time_step
```

Comment:
There must be a better way to do this. Time is one of the most complicated things so if there is no builld in function for it maybe try using a plugin from pip ( dont forget to add it to the  .sh files I created )